opus/application/static_media/js/widgets.js

```diff
@@ -1365,6 +1365,18 @@ var o_widgets = {
             } else {
                 o_search.getHinting(slug);
             }
+
+            // if a RANGE widget that has no qtype & unit inputs just got open, we need to
```

Comment:
Why is this limited to widgets with no qtype and units? Even if the other cases are taken care of in alignDataInSelectionsAndExtras it's very confusing to have this limitation here with no explanation. Wouldn't it be simpler to just always do it?

Reply:
Just updated it to always check & update opus.selections when input widgets just got opened. Previously I limited to widgets with no qtype & units because opus.selections of other cases will be updated in alignDataInSelectionsAndExtras. But you are right, it's confusing and looks like a special case by only doing this step for widgets with no qtype & units there. Now we do the step (update opus.selections) for all cases of input widgets.